datamodel/include/datamodel/geometry/Mesh.hpp

```diff
@@ -9,9 +9,11 @@
 #include <rcbe-engine/datamodel/math/math_constants.hpp>
 #include <rcbe-engine/datamodel/visual/RGBAColor.hpp>
 
+#include <rdmn-engine/public_api.hpp>
+
 namespace rcbe::geometry {
 
-class Mesh {
+R_PUBLIC_API class Mesh {
```

Comment:
According to this https://learn.microsoft.com/en-us/cpp/cpp/using-dllimport-and-dllexport-in-cpp-classes?view=msvc-170
This notation 
1. Is wrong
2. If done right, will export all members